app/messaging/create-message-sender.js

```diff
@@ -0,0 +1,30 @@
+const { MessageSender } = require('ffc-messaging')
+
+const cachedSenders = {}
+
+const createMessageSender = (config) => {
+  if (cachedSenders[JSON.stringify(config)]) {
+    return cachedSenders[JSON.stringify(config)]
```

Comment:
maybe could use the queue name as the key rather than the whole stringified JSON?